scout/adapter/mongo/event.py

```diff
@@ -294,7 +294,7 @@ def add_phenotype(
         if hpo_term:  # User specified an HPO term
             hpo_results = [hpo_term]
         elif omim_term:  # User specified an OMIM diagnosys, collect its associated HPO terms
-            disease_obj = self.disease_term(omim_term)
+            disease_obj = self.disease_term(omim_term, filter_project={"genes": 0})
```

Comment:
HPO terms are needed here

Reply:
But are they really? This function is quite weird, because until last week OMIM terms didn't have associated phenotypes, and how come the event worked anyway?

Reply:
Nah, this function is not invoked by passing OMIM terms, only HPO terms, but there is an associated test using OMIM terms, so I'll just leave it like this

Reply:
Good suggestion, you just need those in the following lines!